test/evergreen.yml

```diff
@@ -231,6 +231,15 @@ tasks:
         vars:
           posix_configure_flags: --enable-silent-rules --enable-diagnostic --enable-strict --enable-python
 
+  # Compile pcc by enabling strict, python, diagnostic and snappy flags
+  - name: compile-ppc
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ PATH=/opt/mongodbtoolchain/v3/bin:$PATH CFLAGS="-ggdb -fPIC"
+          posix_configure_flags: --enable-strict --enable-python --enable-diagnostic --enable-snappy
+          
```

Comment:
Instead of doing an apple-to-apple migration for this job, I'd prefer to understand the importance of the configure differences compared to existing `unit-test` task, see if we can come out a standard configure for unit tests (Python test suite). 
@hbokhari is doing similar investigation for his `wiredtiger-test-unit` job.